addons/website/static/tests/tours/snippet_version.js

```diff
@@ -0,0 +1,71 @@
+odoo.define("website.tour.snippet_version", function (require) {
+"use strict";
+
+var tour = require("web_tour.tour");
+
+tour.register("snippet_version", {
+    test: true,
+    url: "/",
+}, [{
+    content: "Enter edit mode",
+    trigger: 'a[data-action=edit]',
+}, {
+    content: "Drop s_carousel snippet",
+    trigger: '#oe_snippets .oe_snippet:has(.s_carousel) .oe_snippet_thumbnail',
+    run: "drag_and_drop #wrap",
+}, {
+    content: "Drop s_carousel snippet",
+    trigger: '#oe_snippets .oe_snippet:has(.s_text_image) .oe_snippet_thumbnail',
+    run: "drag_and_drop #wrap",
+}, {
+    content: "Test t-snippet and t-snippet-call: snippets have data-snippet set",
+    trigger: '#oe_snippets .o_panel_body > .oe_snippet.ui-draggable',
+    run: function () {
+        // Tests done here as all these are not visible on the page
+        const draggableSnippets = document.querySelectorAll('#oe_snippets .o_panel_body > .oe_snippet.ui-draggable > :nth-child(2)');
+        if (![...draggableSnippets].every(el => el.dataset.snippet)) {
+            tour._consume_tour(tour.running_tour, 'Some t-snippet are missing their template name');
```

Comment:
Really strange, I think we can simply log and error starting by 'error' to make the tour fail or something like that ?